### PR TITLE
Replace wc -l with stdout_lines | length

### PR DESCRIPTION
**Pull request by elfiesmelfie**
Updated roles:
* test_alert
* test_snmp_trap
* test_metrics_retention
* test_sensubility
* test_verify_email

### CODE REVIEW

**Review by mgirgisf — DISMISSED** [review]
thanks emma

File: roles/test_sensubility/tasks/test_health_status.yml [hunk 1]
```diff
@@ -71,9 +66,8 @@
     cmd: >-
       /usr/bin/curl -k {{ prom_auth_string }} \
         -g https://{{ prom_url }}/api/v1/query? \
-        --data-urlencode 'query=last_over_time(sensubility_container_health_status{process="logrotate_crond",host="controller-0.redhat.local"}[10m])' \
-        | grep -oP '(?<="value":).*' | awk -F, '{ print $2 }' |  grep -o '[0-9]\+' | grep 1 | wc -l
+        --data-urlencode 'query=last_over_time(sensubility_container_health_status{process="logrotate_crond",host="ceph-0.redhat.local"}[10m])' \
```
Comment (vkmc) [inline roles/test_sensubility/tasks/test_health_status.yml:69]: This change seems unrelated

**Review by vkmc — CHANGES_REQUESTED** [review]
Not sure if the change in the host for the RHELOSP-176035 is intended, but it seems unrelated. Maybe this comes from a different change?

File: roles/test_sensubility/tasks/test_health_status.yml (hunk 1 above)
Comment (elfiesmelfie) [reply roles/test_sensubility/tasks/test_health_status.yml:69]: This comes from here: https://github.com/infrawatch/feature-verification-tests/pull/180

I believe this shows up because of the order in which these changes were proposed.

Comment (elfiesmelfie) [reply roles/test_sensubility/tasks/test_health_status.yml:73]: I've targetted this change against PR#180, instead of master. I think that should reflect the correct changes for this PR, as well as represent the dependency

**Review by vkmc — APPROVED** [review]
LGTM, thanks Emma!

File: roles/test_alerts/tasks/test_creating_a_standard_alert_route_in_alert_manager.yml [hunk 2]
```diff
@@ -45,18 +45,19 @@
     cmd: >-
       oc exec -it prometheus-default-0 -c prometheus -- /bin/sh -c 'curl -k -H \
         "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-        https://default-alertmanager-proxy:9095/api/v1/alerts' | grep 'active' | grep 'Collectd metrics receive rate is zero' | wc -l
+        https://default-alertmanager-proxy:9095/api/v1/alerts' | grep 'active' | grep 'Collectd metrics receive rate is zero'
+
```
Comment (elfiesmelfie) [inline roles/test_alerts/tasks/test_creating_a_standard_alert_route_in_alert_manager.yml:49]:
```suggestion
```

**Review by ayefimov-1 — APPROVED** [review]
Looks like better logic.

**Review by lnatapov — APPROVED** [review]
LGTM
